docker-compose.yaml

```diff
@@ -112,6 +112,17 @@ services:
       KMAGIC_ALLOW_TOPIC_DELETE: "true"
       KMAGIC_ALLOW_SCHEMA_DELETE: "true"
 
+  redpanda:
```

Comment:
Should we remove Kafka Magic UI?

Reply:
Redpanda has more features and open source. KafkaMagic has some basic features behind a paywall (like creating and deleting topics), so KafkaMagic is less useful IMO.